test/runner/run.js

```diff
@@ -83,7 +83,19 @@ export async function run( {
 				if ( errors ) {
 					pendingErrors[ reportId ][ message.data.name ] = errors;
 				} else {
-					delete pendingErrors[ reportId ][ message.data.name ];
+					const existing = pendingErrors[ reportId ][ message.data.name ];
+
+					// Show a message for flakey tests
+					if ( existing ) {
+						console.log();
```

Comment:
What's the reason for these empty `console.log`s?

Reply:
I'm going to assume you're okay with that. We can always change in a subsequent PR.